packages/interpolate-components/package.json

```diff
@@ -0,0 +1,32 @@
+{
+	"name": "@automattic/interpolate-components",
+	"version": "1.1.2",
+	"description": "Convert strings into structured React components",
+	"main": "dist/cjs/index.js",
+	"module": "dist/esm/index.js",
+	"calypso:src": "src/index.js",
+	"sideEffects": false,
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Automattic/wp-calypso.git",
+		"directory": "packages/interpolate-components"
+	},
+	"author": "Automattic Inc.",
+	"license": "GPL-2.0-or-later",
+	"bugs": {
+		"url": "https://github.com/Automattic/wp-calypso/issues"
+	},
+	"homepage": "https://github.com/Automattic/wp-calypso/tree/HEAD/packages/interpolate-components#readme",
+	"devDependencies": {
+		"@automattic/calypso-typescript-config": "^1.0.0",
+		"react-dom": "^17.0.2"
+	},
+	"peerDependencies": {
+		"react": "^17.0.2"
```

Comment:
Could this be widened? `React.Fragment` has been available since `16.2`, maybe this could be `>=16.2.0`?.

Reply:
I created https://github.com/Automattic/wp-calypso/pull/57564 for this